freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -143,6 +143,11 @@ def _get_bundles_to_rebuild(self):
         log.debug("Getting NVR mapping of original images to rebuilt images")
         nvrs_mapping = self._create_original_to_rebuilt_nvrs_map()
 
+        if (isinstance(self.event, ManualBundleRebuildEvent) and 
+            self.event.requester_metadata_json.get("bundle_related_image_overrides", False)):
+            log.info("Performing manual NVR remapping")
+            self._manual_nvr_remap(nvrs_mapping)  #inplace substitutions to nvrs_mapping
```

Comment:
I think it's unnecessary to have a method to keep several lines with simple logic, but please add a comment to explain what exact the replacement is doing.

Reply:
Okay! I'll "de-extract" that method and make an explanatory comment